tests/http_test.toit

```diff
@@ -1,249 +0,0 @@
-// Copyright (C) 2018 Toitware ApS. All rights reserved.
```

Comment:
Do we have a test like this for the new http package?

Reply:
There are examples, yes

Reply:
I worry that we get more and more untested functionality in packages and we will have no way of knowing if we're breaking things. Can we add tests to the packages (in this case the http one) and set it up so we run our package tests through GitHub actions?

Reply:
Sure, but sounds a bit out of scope for this change.

Reply:
I can see that - and I am all for getting this cleaned up for users of the Toit SDK. Will you file an issue for the postponed testing, so we don't forget?